src/main/java/org/galatea/starter/domain/MetaData.java

```diff
@@ -0,0 +1,33 @@
+package org.galatea.starter.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import javax.persistence.Entity;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@Entity
+@RequiredArgsConstructor
+
+public class MetaData {
+
+  //Variables to define each element of the meta data
+  private String information;
+  private String symbol;
+  private String lastRefreshed;
+  private String outputSize;
+  private String timeZone;
+
+  public MetaData (
```

Comment:
you similarly don't need this constructor since one is auto-generated for you as part of `@Data` (see above) - the `@JsonProperty` annotations can go directly above their corresponding properties above!